pin bump optimum

Gasoonjia opened the pull request:
Summary: bump up optimum to keep sdpa kernel not being decomposed during lowering to cuda delegate

Differential Revision: D87519599

larryliu0820 reviewed (approved):
Review automatically exported from Phabricator review in Meta.

Gasoonjia committed:
pin bump optimum (#15912)

Summary:

bump up optimum to keep sdpa kernel not being decomposed during lowering to cuda delegate

Reviewed By: larryliu0820

Differential Revision: D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599

Gasoonjia committed:
Merge branch 'main' into export-D87519599
